EIPS/eip-2930.md

```diff
@@ -16,7 +16,7 @@ Adds a transaction type which contains an access list, a list of addresses and s
 
 ## Abstract
 
-We introduce a new [EIP-2718](./eip-2718.md) transaction type, with the format `rlp([1, [chainId, nonce, gasPrice, gasLimit, to, value, data, access_list, senderV, senderR, senderS]])`.
+We introduce a new [EIP-2718](./eip-2718.md) transaction type, with the format `3 || rlp([nonce, gasPrice, gasLimit, to, value, data, access_list, senderV, senderR, senderS])`.
```

Comment:
Any reason to set this back from `1` to `3`? Why `3`?

Reply:
Sorry, was a copy/paste error I think (not intentional).  Though, 2972 tentatively uses ID 0 and 1, so maybe we actually want this to be 2?  I'll switch it to 1 for this PR, but I will create a follow-up to change it to 2.

Reply:
Fixed.  Once this is approved/merged I'll PR a change to `2` so it doesn't conflict with 2972, and I do feel like types 0 & 1 for 2972 is appropriate, since it isn't actually a new transaction type, just wrapper.

Reply:
Agreed.